Update vifm to v0.8

cfillion opened the pull request:
http://sourceforge.net/p/vifm/mailman/message/34282033/

bfontaine commented:
The test is failing:

```
==> /usr/local/Cellar/vifm/0.8/bin/vifm -c :q
Failed to open terminal for output.
Error: vifm: failed
```

cfillion commented:
Fixed. I replaced the test because the original one (executing vifm's `:quit` command) cannot run without a proper terminal and faking it with `script` like in ranger's formula (#39661) doesn't suffice.

I couldn't find a better way to simulate a tty and its size in the test environment.

mikemcquaid commented:
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `vifm 0.8` (no added `v`).
